### ISSUE TITLE
Default template not being picked up

### ISSUE THREAD

**Issue by sjDCC** [DMPRoadmap/roadmap#474]
When creating a plan for Glasgow Uni + No funder, I should get default template as institution doesn't have one. The message tells me this is the case but no set of questions loads. Perhaps default template isn't set?
![default-template](https://user-images.githubusercontent.com/6169940/27614403-5d3f8028-5b99-11e7-94a1-f1b8b73d18c1.PNG)

**Comment by briri** [comment]
Just confirming that this is still an issue and adding to the project board.
checking both the 'no funder' 'no research org' boxes on the create plan page results in the selection of the DCC template but NOT the live published version (selected template id = 3).

Both DCC templates in roadmap-stg (ids 3 and 4) have `is_default` set to true but id 4 is `published`. 
Need to update the logic to use the published version. We may also want to consider updating the versioning logic to set the old record's is_default flag to false

**Comment by xsrust** [comment]
There was no requirement in the logic that the template be published.  I have added this and created PR's for development and CDL-MVP.

**Comment by sjDCC** [comment]
Great, thanks @xsrust Did you also remove the default setting from the unpublished plan?

Unless there's a use case to do otherwise, I guess we only want one template set as 'is_default' and only one version of any given template published at a time.

**Comment by xsrust** [comment]
I did not remove is_default from the un-published plan. as this would need to be a database change to fix the current data, and then some code to ensure that when we publish a new version of the default template, we take away default status from the now unpublished version.

In practice, this should not affect us as I added the requirement that the template be published and default, so since the logic is already in place to ensure that only one version of a template is published at a time, we should be covered.

If desired I can go in and add the logic to ensure that is_default gets removed from a plan when a new version is published (and also to ensure that when there is a version being edited, but not yet published, that it does not have this flag), but functionally I dont think it's required.  @vyruss Thoughts on this?   If we do add that extra piece of code, you'll need to remove the 'is_default' flag from any un-published templated in the DB's for all 3 of our services
